### PR TITLE
Move 5 nodes from oko-op-05(OCP4.14) to oko-op-06(OCP4.15)

### PR DESCRIPTION
**Pull request by lewisdenny**

### CODE REVIEW

**Comment by lewisdenny** [comment]
/hold

This can be merged once [1] merges.

/cc @viroel @frenzyfriday

[1] https://github.com/openshift/release/pull/52358

File: clusters/hosted-mgmt/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-14-vexxhost-hybrid_clusterpool.yaml [hunk 1]
```diff
@@ -20,12 +20,6 @@ spec:
   installAttemptsLimit: 3
   installConfigSecretTemplateRef:
     name: install-config-hybrid-3cp
-  inventory:
```
Comment (viroel) [inline clusters/hosted-mgmt/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-14-vexxhost-hybrid_clusterpool.yaml:23]: You also need to update:
  runningCount: 5
  size: 5
  maxConcurrent: 5
  maxSize: 5

to zero :D

Comment (lewisdenny) [reply clusters/hosted-mgmt/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-14-vexxhost-hybrid_clusterpool.yaml:23]: too much rush to finish :D

Comment (lewisdenny) [reply clusters/hosted-mgmt/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-14-vexxhost-hybrid_clusterpool.yaml:23]: Done :)

**Pull request merged**